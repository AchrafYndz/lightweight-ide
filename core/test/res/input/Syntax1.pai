// Main entry point
fn main() {
    let x = 5;
    return 0;
}