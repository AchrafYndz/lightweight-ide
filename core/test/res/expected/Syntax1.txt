{"function_name":[{"end":[1,6],"start":[1,3]}],"identifier":[{"end":[2,8],"start":[2,8]}],"keyword":[{"end":[1,1],"start":[1,0]},{"end":[2,6],"start":[2,4]},{"end":[3,9],"start":[3,4]}],"literal":[{"end":[2,12],"start":[2,12]},{"end":[3,11],"start":[3,11]}]}